src/absil/ilread.fs

```diff
@@ -1029,377 +1032,373 @@ let count c =
 #endif
         
 
-let seekReadUInt16Adv ctxt (addr: byref<int>) =  
```

Comment:
@ncave I think use ``#if`` for now.  The code is performance sensitive for startup of the F# compiler and it's best not to disturb it. thanks!